cli/command/formatter/disk_usage.go

```diff
@@ -26,7 +26,7 @@ const (
 	uniqueSizeHeader  = "UNIQUE SiZE"
 )
 
-// DiskUsageContext contains disk usage specific information required by the formater, encapsulate a Context struct.
+// DiskUsageContext contains disk usage specific information required by the formatter, encapsulate a Context struct.
```

Comment:
Thanks for your contribution. I saw you fixed that place as well. 
Maybe still something for us to do. Could you squash the two commits in this PR into a single one?